mobile/lib/providers/app_life_cycle.provider.dart

```diff
@@ -46,12 +46,17 @@ class AppLifeCycleNotifier extends StateNotifier<AppLifeCycleEnum> {
 
     // Needs to be logged in
     if (isAuthenticated) {
+      // switch endpoint if needed
+      _ref.read(authProvider.notifier).setOpenApiServiceEndpoint();
```

Comment:
Since `Store.tryGet(StoreKey.autoEndpointSwitching)` returns a future and you would need conditional statement to check for null, I moved it inside the setOpenApiServiceEndpoint to make the call to this function cleaner